index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <title>Mac Calculator</title>
+    <link rel="stylesheet" href="styles.css"/>
+  </head>
+
+  <body>
+
+    <div class="calculator-container">
+      <div class="calculator-display">
+        <div class="calculator-display-port">
+          0
+        </div>
+      </div>
+
+      <div class="calculator-row-1">
```

Comment:
you could change all rows to `calculator-row` and then use `. calculator-row:nth-child(3)` etc. to target these rows.